test/suites/console.sh

```diff
@@ -0,0 +1,22 @@
+test_console() {
+  ensure_import_testimage
+
+  # Test container creation
+  lxc init testimage cons1
+
+  # Test the console log file requests.
+  lxc start cons1
+  # No console log file set so this should return an error.
+  ! lxc console cons1 --show-log
+
+  lxc stop --force cons1
+  # Set a console log file.
+  lxc config set cons1 raw.lxc 'lxc.console.logfile=/var/log/lxd/cons1-console.log'
```

Comment:
This is going to run into problems when running multiple test runs in parallel. The logfile path should be part of the temporary directory used for the test run.

Reply:
done